email/blocker.go

```diff
@@ -26,12 +26,11 @@ type (
 	// Blocker is an object that will periodically scan the database for abuse
 	// reports that have not been blocked yet.
 	Blocker struct {
-		staticBlockerApiUrl     string
-		staticBlockerAuthHeader string
-		staticContext           context.Context
-		staticDatabase          *database.AbuseScannerDB
-		staticLogger            *logrus.Entry
-		staticWaitGroup         sync.WaitGroup
+		staticBlockerApiUrl string
+		staticContext       context.Context
+		staticDatabase      *database.AbuseScannerDB
```

Comment:
This is a bit strange. We have a `database` module which makes it seem that the database is only directly interacted with from there but the blocker also contains a connection.

We should probably clean that up in a f/u.

Reply:
Not sure I'm following. We have a `database` module indeed, and it has `MongoDB` struct. We also expose an `AbuseScannerDB` which wraps a `MongoDB` and exposes methods specific to the abuse scanner's database. The `MongoDB` has methods to ensure the schema etc.